Samples/Block hero banner from Demo/README.md

```diff
@@ -0,0 +1,22 @@
+# Block hero banner from Demo
+
+**Tested in Litium version: 7.2.3**
+
+Source code for implementing the *Hero banner* block from the Litium demo site. This is a full width banner block with a customizable CTA button. 
+
+## Instructions
+
+1. Copy `HeroBannerBlockTemplateSetup.cs` to `\Src\Litium.Accelerator\Definitions\Blocks` and include.
+2. Add translations for the block to `\Src\Litium.Accelerator.Mvc\Site\Resources\Administration\Administration.resx`
+    1. **key**=`fieldtemplate.blockarea.herobanner.name` **value**=`Hero banner`
+    2. **key**=`fieldtemplate.blockarea.herobanner.fieldgroup.general.name` **value**=`General`
```

Comment:
Instead of adding the resources into `Administration.resx` you can add a separate `HearoBannerBlock.resx` in the same folder, then it is only to copy/paste the file.

Reply:
Great suggestion. Made the change in a new commit.